### PR TITLE
CTCP-1909 - fix for extra curly braces at the end of json

### PR DESCRIPTION
**Pull request by sonisn1988**

### CODE REVIEW

**Comment by drnaylor-oc** [comment]
Unit tests please.

File: app/uk/gov/hmrc/transitmovementsvalidator/services/JsonValidationService.scala [hunk 1]
```diff
@@ -71,7 +72,7 @@ trait JsonValidationService {
 
 class JsonValidationServiceImpl @Inject() extends JsonValidationService {
 
-  private val mapper = new ObjectMapper
+  private val mapper = new ObjectMapper().enable(DeserializationFeature.FAIL_ON_TRAILING_TOKENS);
```
Comment (drnaylor-oc) [inline app/uk/gov/hmrc/transitmovementsvalidator/services/JsonValidationService.scala:75]: nitpick: that semicolon doesn't need to be there.

Comment (sonisn1988) [reply app/uk/gov/hmrc/transitmovementsvalidator/services/JsonValidationService.scala]: Removed

**Review by drnaylor-oc — APPROVED** [review]

**Review by liandra-giacomine — CHANGES_REQUESTED** [review]
When testing this I still get 500
{
    "message": "Internal server error",
    "code": "INTERNAL_SERVER_ERROR"
}

**Review by liandra-giacomine — APPROVED** [review]
user error, I ran tested with the wrong service!

**Pull request merged**